pythonforandroid/recipes/libffi/__init__.py

```diff
@@ -37,50 +33,11 @@ def build_arch(self, arch):
                     '--prefix=' + self.get_build_dir(arch.arch),
                     '--disable-builddir',
                     '--enable-shared', _env=env)
-            # '--with-sysroot={}'.format(self.ctx.ndk_platform),
-            # '--target={}'.format(arch.toolchain_prefix),
-
-            # ndk 15 introduces unified headers required --sysroot and
-            # -isysroot for libraries and headers. libtool's head explodes
-            # trying to weave them into it's own magic. The result is a link
-            # failure trying to link libc. We call make to compile the bits
-            # and manually link...
-
-            try:
-                shprint(sh.make, '-j5', 'libffi.la', _env=env)
```

Comment:
It seems that now we got `libffi` working with the first build without linkage problems...so I removed all this code block